modules/standard/Random.chpl

```diff
@@ -89,12 +128,87 @@ record SeedGenerators {
   }
 };
 
-const SeedGenerator: SeedGenerators;
+
+// CHPLDOC FEEDBACK: If easy, I'd suggest either deprecating the 
+// :arg <type> <name>: form or else switching the order to 
+// :arg <name> <type>: as it's an easy trap to simply type the same
+// argument as in the function's signature.
+
+
+// CHPLDOC FIXME: int(64) prints out as int(64)(64)
+
+// CHPLDOC FIXME: if the first line below is shifted one character to the
+// left, it ends up being rendered like a method
+
+// CHPLDOC FIXME: if the first line below is shifted one character to the
+// right, it ends up causing a warning (promoted to error) in the .rst
+// file.  It'd be preferable to have it declare the issue in terms of
+// the .chpl line numbers.
+
```

Comment:
Would you mind emailing me the generated files these result in?  I'm having trouble picturing it